lib/inc/drogon/WebSocketConnection.h

```diff
@@ -112,7 +113,7 @@ class WebSocketConnection
      * @param type The message type.
      */
     virtual void send(
-        const std::string &msg,
+        const std::string_view msg,
```

Comment:
Since a const-qualifier in a function declaration does not affect the signature, we don't need it but need it for function definitions.

Reply:
Do we leave them inconsistent?

Reply:
Yes, function declarations are for users. For references, constness changes the meaning but has no changes for values.

https://clang.llvm.org/extra/clang-tidy/checks/readability/avoid-const-params-in-decls.html